docs/MVS_Outputs.rst

```diff
@@ -279,6 +279,22 @@ As above, we apply a weighting based on Electricity Equivalent.
        Degree of Autonomy = \frac{\sum_{i} E_{demand,i} \cdot w_i - \sum_{j} E_{consumption,provider,j} \cdot w_j}{\sum_{i} E_{demand,i} \cdot w_i}
 
 
+Degree of net zero energy (NZE)
+###############################
+
+The degree of net zero energy describes the overall renewable energy generated locally minus the excess energy divided by the overall energy consumed.
+In NZE systems import and export of energy is allowed while the balance over one year should be zero. If more energy is exported than imported it is plus-energy system.
+
+A degree of NZE lower 1 shows the share/potential of reaching net zero energy status,
+while a degree of NZE of 1 represents a net zero energy system
+and a degree of NZE higher 1 a plus-energy system.
+
+As above, we apply a weighting based on Electricity Equivalent.
+
+.. math::
+        NZE &=\frac{\sum_{i} {E_{RE\_generation} (i) \cdot w_i - E_{excess} (i) \cdot w_i}}{\sum_i {E_{demand} (i) \cdot w_i}}
```

Comment:
```suggestion
        NZE &=\frac{\sum_{i} {E_{RE\_generation,i} \cdot w_i - E_{excess, i} \cdot w_i}}{\sum_i {E_{demand, i} \cdot w_i}}
```
The abbreviation `NZE` is, I think, misleading, as one can read it as `Net Zero Energy=0.2`, instead of `Degree of Net Zero Energy`. 

I think the name may still change... `Balance of renewable self-sufficiency`...?

Reply:
I use `Degree of NZE` in the equation now, as in the other KPIs.
hmm I like `Degree of NZE` as it is a definition while everything with self-sufficiency leaves room for interpretation - bilanziell, nicht bilanziell?